scripts/snakemake_sophie_format_training_compendium.R

```diff
@@ -0,0 +1,39 @@
+library(readr)
+library(dplyr)
+library(tibble)
+
+# this script reads in a Pa compendium and SRA-derived strain annotations for
+# samples in the compendium. It filters the compendium to strain-specific samples
+# and formats it for input to SOPHIE.
+
+# read in compendium and metadata -----------------------------------------
+
+strain <- unlist(snakemake@wildcards[['strain']])
+strain <- toupper(strain) # fix case to match whats recorded in strain metadata tbl
+#strain <- "pa14"
+
+strain_metadata <- read_tsv(snakemake@input[['strain']], show_col_types = F) %>%
+# strain_metadata <- read_tsv("inputs/original_compendia/SRA_annotations.tsv", show_col_types = F) %>%
+  rename(experiment = Experiment, strain_type = "Strain type")
+
+compendium <- read_csv(snakemake@input[['raw_compendium']], show_col_types = F) %>%
+# compendium <- read_csv("inputs/original_compendia/num_reads_pa14_cdna_k15.csv", show_col_types = F) %>%
```

Comment:
```suggestion
strain_metadata <- read_tsv(snakemake@input[['strain']], show_col_types = F) %>%
  rename(experiment = Experiment, strain_type = "Strain type")

compendium <- read_csv(snakemake@input[['raw_compendium']], show_col_types = F) %>%
```
(assuming you want to remove these commented out lines, which may or may not be the case)